model/Service/Result/DecryptResultService.php

```diff
@@ -158,11 +159,18 @@ protected function decryptByResult($deliveryIdentifier, $resultId, &$resultsDecr
             );
 
             foreach ($itemsTestsRefs as $ref) {
-                $variableStoreService->save(
-                    $deliveryResultIdentifier,
-                    $this->getResultRow($ref),
-                    $resultStorage
-                );
+                try {
+                    $variableStoreService->save(
+                        $deliveryResultIdentifier,
+                        $this->getResultRow($ref),
+                        $resultStorage
+                    );
+                } catch (DuplicateVariableException $e) {
+                    $this->logInfo(sprintf(
+                        'Variable for result `%s` already exists. Synchronisation of that variable was skipped.',
```

Comment:
are you sure that taoEncryption has to be responsible for the Synchronization process? :(

Reply:
maybe it can be `Decryption of that variable`?